build.gradle

```diff
@@ -250,10 +250,10 @@ dependencies {
     implementation "org.unbescape:unbescape:1.1.6.RELEASE"
     implementation "org.slf4j:slf4j-api:1.7.25"
 
-    testImplementation 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.13'
```

Comment:
Latest JUnit version is 4.13.2. Any particular reason you’re not updating to the latest version?